.drone.star

```diff
@@ -305,6 +322,7 @@ def acceptance():
 		'logLevel': '2',
 		'federatedServerNeeded': False,
 		'federatedServerVersion': '',
+		'runningOnOCIS': False,
```

Comment:
can we set that variable to default false?

Reply:
Yes, That is exactly what we are doing here.